### PR TITLE
Add any missing channel capacity to reserve

### PR DESCRIPTION
**Pull request by benthecarman**
To avoid confusion like in https://github.com/MutinyWallet/mutiny-web/issues/863 lets just add any remaining capacity to the reserve

### CODE REVIEW

File: mutiny-core/src/nodemanager.rs [hunk 1]
```diff
@@ -142,13 +142,22 @@ pub struct MutinyChannel {
...
     fn from(c: &ChannelDetails) -> Self {
+        let size = c.channel_value_satoshis;
+        let balance = c.next_outbound_htlc_limit_msat / 1_000;
+        let inbound = c.inbound_capacity_msat / 1_000;
+        let mut reserve = ((c.outbound_capacity_msat - c.next_outbound_htlc_limit_msat) / 1_000)
+            + c.unspendable_punishment_reserve.unwrap_or(0);
+
+        // if there is any diff in the channel size and the other totals,
+        // just add it to the reserve to avoid confusion
+        reserve += size - (balance + inbound + reserve);
```
Comment (TonyGiorgio) [inline mutiny-core/src/nodemanager.rs:153]: I don't understand this. You're adding to reserve an amount that subtracts the reserve. I'd like to see some numbers / tests for these calculations.

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]

File: mutiny-core/src/nodemanager.rs (hunk 1 above)
Comment (benthecarman) [reply mutiny-core/src/nodemanager.rs]: Okay simplified

**Review by TonyGiorgio — APPROVED** [review]
i have absolutely no idea if this is correct and there's no tests but yolo i guess

**Comment by TonyGiorgio** [comment]
needs rebase

**Pull request merged**